Sources/idt/idt.cc

```diff
@@ -67,14 +76,100 @@ const std::set<std::string> &get_ignored_symbols() {
 }
 
 namespace idt {
+struct IncludeCollector : clang::PPCallbacks {
```

Comment:
Why `IncludeCollector`? It can be re-used right? Perhaps `PPCallbacks` or `PreprocessingHandler`?